newtests/bigint/test.js

```diff
@@ -201,156 +201,7 @@ export default suite(({ addFile, addFiles, addCode }) => [
       const valid_neg_small = -100n;
       type ValidSmall = 100n;
       type ValidNegSmall = -1n;
```

Comment:
Tests for BigInts with numeric separators are missing here. Parser already support them.